packages/wallet/api/src/lib-wallet.js

```diff
@@ -178,6 +178,7 @@ export function makeWalletRoot({
   const brandToAutoDepositPurse = makeScalarMapStore('brand');
 
   // Offers that the wallet knows about (the inbox).
+  /** @type {MapStore<Passable, any>} */
```

Comment:
id has to be a string. 
```suggestion
  /** @type {MapStore<string, any>} */
```